README.md

```diff
@@ -261,15 +261,8 @@ package managers:
 - `gem.txt`: run-time dependencies installed by gem.
 - `npm.txt`: run-time dependencies installed by npm.
 - `pip.txt`: a normal [pip `requirements.txt`][] file, for run-time
-  dependencies too.
-
-All of these files will install separately each line. See this example
```

Comment:
It is no longer true after #57. I forgot to remove it then. All is installed at once, benefiting from each package manager cache and dependency resolution system.